src/core/meshcop/commissioner.cpp

```diff
@@ -79,14 +79,25 @@ Commissioner::Commissioner(ThreadNetif &aThreadNetif):
     mNetif(aThreadNetif)
 {
     memset(mJoiners, 0, sizeof(mJoiners));
+}
+
+otInstance *Commissioner::GetInstance(void)
+{
+    return mNetif.GetInstance();
+}
+
+void Commissioner::AddCoapResources()
+{
     mNetif.GetCoapServer().AddResource(mRelayReceive);
     mNetif.GetCoapServer().AddResource(mDatasetChanged);
     mNetif.GetSecureCoapServer().AddResource(mJoinerFinalize);
 }
 
-otInstance *Commissioner::GetInstance(void)
+void Commissioner::RemoveCoapResources()
```

Comment:
nit: All void functions or methods shall explicitly declare and specify the void type keyword.